openshift/auth.config.yaml

```diff
@@ -31,6 +29,8 @@ objects:
   data:
     service-account-secrets: >-
       ewogICJhY2NvdW50cyI6IFsKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAiaWQiOiI1ZGVjNWZkYi0wOWUzLTQ0NTMtYjczZi01YzgyODgzMmIyOGUiLAogICAgICAgICAgInNlY3JldHMiOlsid2l0c2VjcmV0Il0KICAgICAgfSwKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXRlbmFudCIsCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgInNlY3JldHMiOlsidGVuYW50c2VjcmV0T2xkIiwgInRlbmFudHNlY3JldE5ldyJdCiAgICAgIH0KICBdCn0=  
+    oso-clusters: >-
+      ewogICJhY2NvdW50cyI6IFsKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAiaWQiOiI1ZGVjNWZkYi0wOWUzLTQ0NTMtYjczZi01YzgyODgzMmIyOGUiLAogICAgICAgICAgInNlY3JldHMiOlsid2l0c2VjcmV0Il0KICAgICAgfSwKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXRlbmFudCIsCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgInNlY3JldHMiOlsidGVuYW50c2VjcmV0T2xkIiwgInRlbmFudHNlY3JldE5ldyJdCiAgICAgIH0KICBdCn0=
```

Comment:
This is the same value as service-account-secrets.  Perhaps change it to this:

`ew0KICAgICJjbHVzdGVycyI6IFsNCiAgICAgICAgew0KICAgICAgICAgICAgIm5hbWUiOiJ1cy1lYXN0LTIiLA0KICAgICAgICAgICAgInVybCI6Imh0dHBzOi8vYXBpLnN0YXJ0ZXItdXMtZWFzdC0yLm9wZW5zaGlmdC5jb20iLA0KICAgICAgICAgICAgInNlcnZpY2UtYWNjb3VudC10b2tlbiI6ImZYMG5IM2Q2OExRNlNLNXdCRTZRZUtKNlg4QVpHVlFPM2RHUVpaRVRha2htZ21XQXFyMktERlhFNjVLVXdCTzY5YVdvcSIsDQogICAgICAgICAgICAidG9rZW4tcHJvdmlkZXItaWQiOiJmODY3YWMxMC01ZTA1LTQzNTktYTBjNi1iODU1ZWNlNTkwOTAiLA0KICAgICAgICAgICAgImF1dGgtY2xpZW50LWlkIjoiYXV0aGVhc3QyIiwNCiAgICAgICAgICAgICJhdXRoLWNsaWVudC1zZWNyZXQiOiJhdXRoZWFzdDJzZWNyZXQiLA0KICAgICAgICAgICAgImF1dGgtY2xpZW50LWRlZmF1bHQtc2NvcGUiOiJ1c2VyOmZ1bGwiDQogICAgICAgIH0sDQogICAgICAgIHsNCiAgICAgICAgICAgICJuYW1lIjoidXMtZWFzdC0yYSIsDQogICAgICAgICAgICAidXJsIjoiaHR0cHM6Ly9hcGkuc3RhcnRlci11cy1lYXN0LTJhLm9wZW5zaGlmdC5jb20iLA0KICAgICAgICAgICAgInNlcnZpY2UtYWNjb3VudC10b2tlbiI6ImFrNjFUNlJTQWFjV0ZydWgxdlpQOGN5VU9CdFEzQ2h2MXJkT0JkZFN1YzluWjJ3RWNzODFESFhSTzU1TnBJcFZROHVpSCIsDQogICAgICAgICAgICAidG9rZW4tcHJvdmlkZXItaWQiOiI4ODZjN2VhMy1lZjk3LTQ0M2QtYjM0NS1kZTk0Yjk0YmI2NWQiLA0KICAgICAgICAgICAgImF1dGgtY2xpZW50LWlkIjoiYXV0aGVhc3QyYSIsDQogICAgICAgICAgICAiYXV0aC1jbGllbnQtc2VjcmV0IjoiYXV0aGVhc3QyYXNlY3JldCIsDQogICAgICAgICAgICAiYXV0aC1jbGllbnQtZGVmYXVsdC1zY29wZSI6InVzZXI6ZnVsbCINCiAgICAgICAgfQ0KICAgIF0NCn0=`